src/translations/de.translation

```diff
@@ -1,1361 +1,1361 @@
+        <label>Kursbeschreibung – Kopierauftrag mit folgendem Status abgeschlossen:</label>
     </customLabels>
     <customLabels>
-        <label>Standardkontakt – Sprachkompetenz</label>
         <name>stgDefaultContactLanguageFluency</name>
+        <label>Standardkontakt – Sprachkompetenz</label>
     </customLabels>
     <customLabels>
-        <label>Standard-Datensatztyp für Lehrpersonal</label>
         <name>stgDefaultFacultyTypeTitle</name>
+        <label>Standard-Datensatztyp für Lehrpersonal</label>
     </customLabels>
     <customLabels>
-        <label>Standard-Datensatztyp für aktive Studenten</label>
         <name>stgDefaultStudentTypeTitle</name>
+        <label>Standard-Datensatztyp für aktive Studenten</label>
     </customLabels>
     <customLabels>
-        <label>Fehlerbehandlung deaktivieren</label>
         <name>stgDisableErrorHandlingTitle</name>
+        <label>Fehlerbehandlung deaktivieren</label>
     </customLabels>
     <customLabels>
-        <label>Erzwingung von &quot;Bevorzugte E-Mail-Adresse&quot; deaktivieren</label>
         <name>stgDisablePreferredEmailEnforcement</name>
+        <label>Erzwingung von "Bevorzugte E-Mail-Adresse" deaktivieren</label>
     </customLabels>
     <customLabels>
-        <label>Erzwingung von &quot;Bevorzugte Telefonnummer&quot; deaktivieren</label>
         <name>stgDisablePreferredPhoneEnforcement</name>
+        <label>Erzwingung von "Bevorzugte Telefonnummer" deaktivieren</label>
     </customLabels>
     <customLabels>
-        <label><!-- EDA Setup title --></label>
         <name>stgEDASetupTitle</name>
+        <label>EDA-Einrichtung</label>
     </customLabels>
     <customLabels>
-        <label>Kursverbindungen aktivieren</label>
         <name>stgEnableCourseConnectionsTitle</name>
+        <label>Kursverbindungen aktivieren</label>
     </customLabels>
     <customLabels>
-        <label>Verbesserte Funktionalität für &quot;Bevorzugte Telefonnummer&quot; aktivieren</label>
         <name>stgEnablePreferredPhoneSync</name>
+        <label>Verbesserte Funktionalität für "Bevorzugte Telefonnummer" aktivieren</label>
     </customLabels>
     <customLabels>
-        <label>Fehlerbenachrichtigungen an:</label>
         <name>stgErrorNotiTo</name>
+        <label>Fehlerbenachrichtigungen an:</label>
     </customLabels>
     <customLabels>
-        <label>Empfänger von Fehlerbenachrichtigungen</label>
         <name>stgErrorNotifRecipientsTitle</name>
+        <label>Empfänger von Fehlerbenachrichtigungen</label>
     </customLabels>
     <customLabels>
-        <label>Abgleich von Rasse oder ethnischer Zugehörigkeit für Kontakte</label>
         <name>stgEthnicityRaceBackfillContacts</name>
+        <label>Abgleich von Rasse oder ethnischer Zugehörigkeit für Kontakte</label>
     </customLabels>
     <customLabels>
-        <label>Fließend</label>
         <name>stgFluent</name>
+        <label>Fließend</label>
```

Comment:
This field should not be translated. It is a system field used only in the install script and refers to an API name.
```suggestion
        <label>Fluent</label>
```